package.json

```diff
@@ -59,6 +59,7 @@
     "husky:pre-commit": "lint-staged && npm run spellcheck",
     "lint": "bin/otomi validate-templates",
     "lint:all": "run-p spellcheck lint",
+    "postinstall": "husky install",
```

Comment:
I don't think this has any effect after having done "npm install", as that registers the hooks already.